src/ert/libc/mman.c

```diff
@@ -142,3 +142,13 @@ int ert_munmap(void* addr, size_t length)
 
     return result;
 }
+
+const void* ert_get_heap_base(void)
+{
+    return __oe_get_heap_base();
+}
+
+size_t ert_get_heap_size(void)
+{
+    return __oe_get_heap_size();
```

Comment:
My understanding was that mmap as implemented by ert_mmap is the whole enclave_heap-sizeof(bitmap)? (Which is `__size`)
Hence, I thought the size difference is negligible,

Reply:
Agreed. Wrote this comment before reviewing the other PR and thought it could lead to bugs. Should we then remove these functions and you just make __oe_get_heap_xxx available by declaring them in ego?

Reply:
Sounds good to me